src/web/mjs/engine/Settings.mjs

```diff
@@ -65,6 +67,18 @@ export default class Settings {
             input: types.directory,
             value: path.join( docs, 'Mangas' )
         };
+        let electron = require( 'electron' );
+        this.bookmarkDirectory = {
+            label: 'Bookmarks Directory ⁽¹⁾',
+            description: [
+                'The directory where bookmarks file will be stored',
+                '',
+                '⁽¹⁾ Restart required to take affect',
+            ].join('\n'),
+            input: types.directory,
+            value: path.join( electron.remote.app.getPath( 'userData' ) )
```

Comment:
use `this.app` instead of `electron.remote.app`

Reply:
The statement `path.join` seems obsolete if there is only a single parameter assigned => nothing to join

Reply:
you mean `app` cause `this.app` is not defined

Reply:
Yes you IDE is correct, its local variable